string action
uint8 target
---
bool success
